ModularBungalow/altjobtitles/alt_titles.dm

```diff
@@ -7,10 +7,15 @@
 
 //Engineering
 /datum/job/engineer
-	alt_titles = list("Maintenance Technician", "Engine Technician", "Electrician", "Signal Technician")
+	alt_titles = list("Maintenance Technician", "Engine Technician", "Electrician")
 	senior_title = list("Project Co-ordinator")
 	ultra_senior_title = list("Bob the Builder")
 
+/datum/job/signal_tech
+	alt_titles = list("DevOps")
+	senior_title = list("Power Coder")
+	ultra_senior_title = list("CentComm Systems Engineer")
```

Comment:
```suggestion
	ultra_senior_title = list("CentCom Systems Engineer")
```

Reply:
wow not making it a meme name like the true ultra senior titles.. shaaaaaame